pipelines/internal/create-advisory-oci-artifact/create-advisory-oci-artifact.yaml

```diff
@@ -0,0 +1,49 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: create-advisory-oci-artifact
```

Comment:
Is the "create-advisory-oci-artifact" pipeline expected to added or just for testing?